server/heroku/logs.go

```diff
@@ -13,18 +13,28 @@ type PostLogs struct {
 	*empire.Empire
 }
 
+type PostLogsForm struct {
+	Duration string
```

Comment:
On a similar note to https://github.com/remind101/kinesumer/pull/22/commits/11e496a3d697d12c961ea2307dc0e97ccbdbd28c#r62950197. I think it might be better if we do the `time.ParseDuration` at the edge of user input (in the emp client) and make this parameter be an int (milliseconds/seconds).